### PR TITLE
Update chklist

### PR DESCRIPTION
**Pull request by rasmus-madsen**
this takes care of the documentation action items from the aes v2 review #10757 
after #11219 is merged 
this PR takes AES dv to V2 and updates the checklist

### CODE REVIEW

File: hw/ip/aes/doc/checklist.md [hunk 1]
```diff
@@ -174,27 +174,27 @@ Review        | [V2_CHECKLIST_SCOPED][]               | Done        |
...
 Testbench     | [ALL_ASSERTION_CHECKS_ADDED][]          | Done        |
 Testbench     | [SIM_TB_ENV_COMPLETED][]                | Done        |
 Tests         | [SIM_ALL_TESTS_PASSING][]               | Done        |
-Tests         | [FPV_ALL_ASSERTIONS_WRITTEN][]          | Not Started |
-Tests         | [FPV_ALL_ASSUMPTIONS_REVIEWED][]        | Not Started |
-Tests         | [SIM_FW_SIMULATED][]                    | Not Started |
-Regression    | [SIM_NIGHTLY_REGRESSION_V2][]           | In Progress |
+Tests         | [FPV_ALL_ASSERTIONS_WRITTEN][]          | N/A         |
+Tests         | [FPV_ALL_ASSUMPTIONS_REVIEWED][]        | N/A         |
+Tests         | [SIM_FW_SIMULATED][]                    | In Progress |
```
Comment (weicaiyang) [inline hw/ip/aes/doc/checklist.md:186]: this should be N/A. It only applies to ibex

File: hw/ip/aes/doc/checklist.md [hunk 2]
```diff
@@ -174,27 +174,27 @@ Review        | [V2_CHECKLIST_SCOPED][]               | Done        |
...
+Regression    | [SIM_NIGHTLY_REGRESSION_V2][]           | Done        |
 Coverage      | [SIM_CODE_COVERAGE_V2][]                | In Progress |
```
Comment (weicaiyang) [inline hw/ip/aes/doc/checklist.md:188]: looks like code coverage has not reached 90%+

**Comment by weicaiyang** [comment]
can we make the branch and expression coverage pass 90%?

**Comment by weicaiyang** [comment]
> can we make the branch and expression coverage pass 90%?

Actually the coverage is way over 90%. But due to https://github.com/lowRISC/opentitan/pull/12219, it doesn't reflect the correct numbers.

**Review by weicaiyang — APPROVED** [review]
LGTM

File: hw/ip/aes/data/aes_testplan.hjson [hunk 3]
```diff
@@ -47,8 +47,8 @@
     {
       name: back2back
       desc: '''
-           Randomly select the spacing between consecutive messages and blocks from 0 - n clock cycles.
-           The distribution will be weighted toward no and small gaps (0-10 cycles) but will also cover larger gaps.'''
+           Back to back blocks are not possible as the DUT need to be idle before writing a new configuration.
```
Comment (vogelpi) [inline hw/ip/aes/data/aes_testplan.hjson:50]: This new description is not accurate:
- Back2Back blocks can very well be done. All that's needed is to read the STATUS register and check the INPUT_READY flag. This definitely needs to be tested and I would assume the current environment already covers this. Is that correct @rasmus-madsen ?
- Back2Back *messages* cannot be done as the DUT needs to be idle before writing a new configuration.

**Review by vogelpi — APPROVED** [review]
This looks good modulo the description of the `back2back` test which isn't accurate anymore with this PR. This should be corrected @rasmus-madsen .

**Comment by weicaiyang** [comment]
@rasmus-madsen looks like this PR is ready to land. Can you fix the conflict and merge it?

**Comment by vogelpi** [comment]
When fixing the conflict, please also revise the description of the back2back test as I pointed out some time ago (https://github.com/lowRISC/opentitan/pull/10763#discussion_r854842413).

File: hw/ip/aes/data/aes_testplan.hjson (hunk 3 above)
Comment (rasmus-madsen) [reply hw/ip/aes/data/aes_testplan.hjson:50]: I think I must have been asleep when I updated this text.
Back2back blocks are indeed verified.
I think all I wanted to say was that back to back messages are not possible as the configuration can not be changed while the DUT is Active

Comment (vogelpi) [reply hw/ip/aes/data/aes_testplan.hjson]: This looks good, thanks for adjusting the description @rasmus-madsen !

**Pull request merged**